vpr/src/route/route_common.cpp

```diff
@@ -494,7 +494,7 @@ void init_route_structs(int bb_factor) {
     route_ctx.net_status.resize(cluster_ctx.clb_nlist.nets().size());
 }
 
-t_trace* update_traceback(t_heap* hptr, ClusterNetId net_id) {
+t_trace* update_traceback(t_heap* hptr, int target_pin, ClusterNetId net_id) {
```

Comment:
Rename and explain target_pin in the comment.  Also, we've changed style to comments before function so better to move the comment above the function.

Reply:
Ah, so this is an odd one out with comments inside the function brackets. I was wondering why it looked a bit different :)
Done changing parameter name and adding/fixing comment.